develop/tutorials/articles/260-themes-and-layout-templates/07-using-developer-mode-with-themes.markdown

```diff
@@ -110,23 +111,41 @@ Once, you've configured your server, add the following properties to your
     layout.template.cache.enabled=false
     minifier.enabled=false
     theme.css.fast.load=false
-    
+ 
 This takes care of most of the Developer Mode properties, but you still have one 
 left to configure. This is covered next.
 
+<!-- Hey Mike, I don't think these instructions are correct. If you look in a
+bundle's tomcat[version]/webapps/ROOT/WEB-INF/classes folder, there's a
+portal-developer.properties file in there, with all the configuration you need.
+You shouldn't need to create a new one. It's on the classpath already. 
+
+The other thing is that when I use the initBundle Gradle command to pull a
+runtime into a Workspace, it pre-configures the portal-ext.properties file with
+this property: 
+
+include-and-override=portal-developer.properties
+
+This would seem to indicate that the way to do this is the same as how you did
+it for 6.2: by adding the above property to portal-ext.properties. 
+
+So my question is, where did the above instructions come from? I'm not sure
+they're right. Thanks! -Rich -->
+
```

Comment:
@sez11a these instructions were taken straight from our existing 6.2 dev mode docs. Since the ticket said that customers were able to follow the existing directions to enable developer mode in 7.0, I assumed they were correct. I apologize for the oversight. I'll update the docs based on your feedback and @blzaugg 's updates. thx!